.evergreen/auth_aws/lib/ecs_hosted_test.js

```diff
@@ -31,7 +31,9 @@ const smoke = runProgram(program, uri);
 assert.eq(smoke, 0, "Could not auth with smoke user");
```

Comment:
Can you change "Could not auth with smoke user" to "driver .evergreen/run-mongodb-aws-ecs-test.sh script failed"?